diff -urp a/_test/roundtrip.py b/_test/roundtrip.py
--- a/_test/roundtrip.py	2018-07-30 09:55:23.619098795 +0100
+++ b/_test/roundtrip.py	2018-07-30 09:55:55.314099405 +0100
@@ -7,7 +7,7 @@ helper routines for testing round trip o
 """
 import sys
 import textwrap
-from ruamel.std.pathlib import Path
+from pathlib import Path
 
 enforce = object()
 
diff -urp a/_test/test_api_change.py b/_test/test_api_change.py
--- a/_test/test_api_change.py	2018-07-30 09:55:23.617098795 +0100
+++ b/_test/test_api_change.py	2018-07-30 09:56:17.258099827 +0100
@@ -9,7 +9,7 @@ testing of anchors and the aliases refer
 import sys
 import textwrap
 import pytest
-from ruamel.std.pathlib import Path
+from pathlib import Path
 
 
 class TestNewAPI:
